### PR TITLE
[9.0][IMP] partner_financial_risk: Improve multicompany cron

### PR DESCRIPTION
**Pull request by carlosdauden**
Force company to match property_account_receivable_id 

@Tecnativa

### CODE REVIEW

File: partner_financial_risk/models/res_partner.py [hunk 1]
```diff
@@ -249,12 +250,24 @@ def process_unpaid_invoices(self):
         ConfigParameter = self.env['ir.config_parameter']
         last_check = ConfigParameter.get_param(
             'partner_financial_risk.last_check', default='2016-01-01')
-        move_lines = self.env['account.move.line'].search([
-            ('reconciled', '=', False),
-            ('account_id.internal_type', '=', 'receivable'),
-            ('date_maturity', '>=', last_check),
-            ('date_maturity', '<', max_date)])
-        move_lines.mapped('partner_id')._compute_risk_account_amount()
+        groups = self.env['account.move.line'].read_group(
```
Comment (pedrobaeza) [inline partner_financial_risk/models/res_partner.py]: Shouldn't this be process with `sudo()`?

Comment (carlosdauden) [reply partner_financial_risk/models/res_partner.py]: Cron execs this process with admin user, but I don't have any problem to add sudo

Comment (pedrobaeza) [reply partner_financial_risk/models/res_partner.py]: Being a public method that can be call from any place, I think it's convenient to be executed with `sudo`

**Comment by carlosdauden** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]
@sergio-teruel please review

**Review by yajo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please bump module version number directly, @Yajo

**Pull request merged**